File: Sort/42746.py
def solution(numbers: list) -> str:
    lists = []
    answer = ""
    for i in numbers:
        lists.append([str(i),str(i)*3])
    lists.sort(reverse=True, key=lambda x: x[1])
    for item in lists:
        answer += item[0]

    return str(int(answer))

solution([6, 10, 2])
solution([3, 30, 34, 5, 9])

File: Javascript/Sort/42746.js
function solution(numbers) {
    let answer = [];
    let result = []; 
    let temp = [];
    for(let number of numbers){
        for(let i=0;i<3;i++){
            temp.push(number);
        }
        result.push({'sort_num':temp.join(''), 'real': number});
                    
        temp = [];
    }
    
    // 정렬 할 변수 기준 정렬
    result.sort((a, b) => {
        return b.sort_num < a.sort_num ? -1 : a.sort_num < b.sort_num ? 1 : 0;
    });
    
    for ( let j of result){
        answer.push(String(j.real));
    }  

    // Every 함수를 통해 number 값이 전부 '0' 이라면 0이 반환 되게함.
    return answer = answer.every(v => v == '0') ? '0' : answer.join('');
}